disnake/client.py

```diff
@@ -346,23 +346,40 @@ def __init__(
         *,
         asyncio_debug: bool = False,
         loop: Optional[asyncio.AbstractEventLoop] = None,
-        **options: Any,
+        shard_id: Optional[int] = None,
+        shard_count: Optional[int] = None,
+        enable_debug_events: bool = False,
+        connector: Optional[aiohttp.BaseConnector] = None,
+        proxy: Optional[str] = None,
+        proxy_auth: Optional[aiohttp.BasicAuth] = None,
+        assume_unsync_clock: bool = True,
+        max_messages: Optional[int] = 1000,
+        application_id: Optional[int] = None,
+        heartbeat_timeout: float = 60.0,
+        guild_ready_timeout: float = 2.0,
+        allowed_mentions: Optional[AllowedMentions] = None,
+        activity: Optional[BaseActivity] = None,
+        status: Optional[Union[Status, str]] = None,
+        intents: Intents = None,
+        chunk_guilds_at_startup: Optional[bool] = None,
+        member_cache_flags: MemberCacheFlags = None,
+        cache_application_command_permissions: bool = True,
```

Comment:
this parameter was removed in 4873114daa57095ed58b944578bf23a29cb7e8f7

.. maybe removing `**kwargs` from state would actually be a good idea (ignore [this](https://github.com/DisnakeDev/disnake/pull/371#issuecomment-1063146371) :^)), if you're alright with making that change in this PR as well. It's perfectly fine if not though, then we'd just make that change in a future PR

Reply:
Yeah, can do, will probably be sometime later this week, though. I'll try to have it done tomorrow.

Reply:
Done in 23194a6fb17a6c8f8e2a8f1a7356152c021cbaa8.